app/mailers/password_reset_mailer.rb

```diff
@@ -0,0 +1,16 @@
+class PasswordResetMailer < ActionMailer::Base
+  # Input:
+  #   user: chef object returned by server
+  def password_reset(user)
+    @user = user
```

Comment:
Is there a reason all the values in this method are set to instance vars? I assume rails has a reason, b/c I'm not seeing why that is the case just based on this method.